### DISCUSSION TITLE
Production Build output

### DISCUSSION THREAD

**Discussion by kewlui-mk** [codesandbox/sandpack#1109] [General]
Is there any way of getting the (npm run build) production build output via Sandpack?

What I would love to do is ideally get a minified js/css output of an App somehow via Sandpack 
(I don't want to build it locally for safety concerns that you have already fixed)
Is this possible in any way?

**Comment by batuhanbilginn** [comment]
Hey there, did you find a solution for this?

**Comment by CompuIves** [comment]
In this case, do you want to get the output of what Sandpack generated in-browser? I don't think we can do that easily, because Sandpack compiles the code specifically to run in our in-browser bundler, so running it locally would be hard to do. We _could_ share the cache, so Sandpack starts it faster.

**Reply by batuhanbilginn** [reply]
What I need is accessing shell to run build command. I want to build my vite app and get the static files.
